FetchMigration/Dockerfile

```diff
@@ -18,5 +18,6 @@ ENV PATH=/root/.local:$PATH
 RUN echo "ssl: false" > $DATA_PREPPER_PATH/config/data-prepper-config.yaml
 RUN echo "metricRegistries: [Prometheus]" >> $DATA_PREPPER_PATH/config/data-prepper-config.yaml
 
-# Include the -u flag to have stdout logged
-ENTRYPOINT python3 -u ./fetch_orchestrator.py $DATA_PREPPER_PATH $FM_CODE_PATH/input.yaml http://localhost:4900
+# Include the -u flag to have unbuffered stdout (for detached mode)
+# "$0" "$@" allows passing extra args/flags to the entrypoint when the image is run
+ENTRYPOINT python3 -u ./fetch_orchestrator.py --insecure $DATA_PREPPER_PATH $FM_CODE_PATH/input.yaml "$0" "$@"
```

Comment:
Are we currently adding any extra args/flags to the entrypoint?

Reply:
Not currently, but given that metadata migration is essential for Traffic Capture & Replay to replicate data accurately, IMO its useful to enable the Docker image to be run with the `--createonly` flag as a pre-requisite step.